brave/brave-ios
# Fix #3280: Build AdblockRust with iOS 12.4 deployment target

Due to a bug with `cargo lipo` on Big Sur, the compilation of adblock-rust-ffi has now been moved to the `update-adblockrust.sh` script



## Summary of Changes


This pull request fixes #3280 

## Submitter Checklist:


## Test Plan:

- Run on iOS 12
- Verify that it does not crash
- Verify adblock stats are tracked

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
